app/connectors/Sub09Connector.scala

```diff
@@ -100,7 +100,7 @@ class Sub09Connector @Inject()(appConfig: AppConfig,
 
     metricsReporter.withResponseTimeLogging("mdg.get.company-information") {
       http.GET[Option[MdgSub09XiEoriInformationResponse]](uri, headers = headers)
-        .map(_.map(v => XiEoriInformation(v.xiEori, v.consent.getOrElse("0"), v.address))).recover {
+        .map(_.map(v => XiEoriInformation(v.xiEori, v.consent.getOrElse("0"), v.address.getOrElse(XiEoriAddressInformation(""))))).recover {
```

Comment:
Could you please create a constant like emptyString in a new object like Utils and use that here as it will make it more readable and better to add a test for the same? If you think it will take time then create a local val emptyString  here in the method itself.

Reply:
created a val with in the function for now. we can refactor it and move into utils as part of a new tech debt ticket along with other changes